public/index.html

```diff
@@ -4,11 +4,11 @@
         <meta charset="utf-8">
         <meta content="IE=edge" http-equiv="X-UA-Compatible" />
         <meta name="viewport" content="width=device-width, user-scalable=no" />
-        <title><%= htmlWebpackPlugin.options.title %></title>
-        <link rel="icon" href="public/favicon.ico">
+        <title>shorebirdie.com</title>
+        <link rel="icon" href="public/logo.png">
```

Comment:
you can remove this link

Reply:
shorebirdie logo teseyala annaya title bar o nundi